applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.h

```diff
@@ -143,13 +145,16 @@ struct broadcast_source_per_adv_data {
  *
  * @param[in]	big_index		Index of the Broadcast Isochronous Group (BIG) to get
  *					advertising data for.
+ * @param[in]	fixed_id		Flag to indicate if the broadcast ID will be random or not.
+ * @param[in]	broadcast_id		Broadcast ID to be used in the advertising data, if
+ *					@p fixed_id is set to true, 3 octets.
```

Comment:
Is it three octets fixed or at maximum, as you write in shell_error(shell, "Broadcast ID must be maximum 3 octets long");, which would suggest it can also be shorter?
```suggestion
 * @param[in]	broadcast_id		Broadcast ID to be used in the advertising data. If
 *					@p fixed_id is set to true, the broadcast ID is long three octets.
```

Reply:
It will always be 3 octets, if you choose to set all the bits is up to you. Meaning: 0x1 will still fill 3 octets and will make the ID 0x000001